### ISSUE TITLE
repository name must be lowercase

### ISSUE THREAD

**Issue by beeman** [gperdomor/nx-tools#6]
Hey @gperdomor - thanks for sharing your Docker build tool.

I just tried it out using your example, but it seems to struggle because my GitHub org name is not in lowercase:

```
invalid argument "docker.pkg.github.com/KikstartDev/kikstart-auth/api:latest" for "-t, --tag" flag: invalid reference format: repository name must be lowercase
```

Would it be possible to force-lowercase this so it works with these orgs too?

**Comment by gperdomor** [comment]
Solved in #15

**Comment by beeman** [comment]
Thanks @gperdomor 👍
